docs/source/api/v5/cdns.rst

```diff
@@ -115,6 +117,7 @@ Request Structure
 :dnssecEnabled: If ``true``, this CDN will use DNSSEC, if ``false`` it will not
 :domainName:    The top-level domain (TLD) belonging to the new CDN
 :name:          Name of the new CDN
+:ttlOverride:	A :abbr:`TTL (Time To Live)` value, in seconds, that, if set, overrides all set set TTL values on :term:`Delivery Services` in this :term:`CDN`
```

Comment:
Same

Reply:
Not repeating `set` in d81596bedf

Reply:
Changed `CDN` to an abbreviation in 53c02be419